doc/nrf/known_issues.rst

```diff
@@ -899,14 +903,12 @@ KRKNWK-15749: Invalid ZAP Tool revision used
        git -C third_party/zap/repo/ checkout -f 2ae226
        git add third_party/zap/repo/
 
-.. rst-class:: v2-1-2 v2-1-1 v2-1-0
```

Comment:
Why is this removed? Btw. the issue is going to be resolved in 2.2 (but final tests are to be made).

Reply:
Fixed.